lib/api/src/modules/stories.ts

```diff
@@ -197,6 +198,20 @@ export const init: ModuleFn = ({
         storiesFailed: error,
       });
     },
+    selectFirstStory: () => {
+      const { storiesHash } = store.getState();
+      const lookupList = Object.keys(storiesHash).filter(
+        (k) => !(storiesHash[k].children || Array.isArray(storiesHash[k]))
+      );
```

Comment:
Can we just use a `find` here rather than filtering the whole list?

Reply:
Yeah that makes sense 👍  I just blindly copied the logic from the other function.

Reply:
https://github.com/storybookjs/storybook/pull/11360/commits/5e1e0908668e0f4d50ecfe11afc7e29e1b7d86d3

Reply:
Heh yeah I figured :)